scripts/linux/runE2ETest.sh

```diff
@@ -1070,14 +1076,16 @@ TRANSPORT_TYPE_1="${TRANSPORT_TYPE_1:-amqp}"
 TRANSPORT_TYPE_2="${TRANSPORT_TYPE_2:-amqp}"
 TRANSPORT_TYPE_3="${TRANSPORT_TYPE_3:-mqtt}"
 TRANSPORT_TYPE_4="${TRANSPORT_TYPE_4:-mqtt}"
-TWIN_UPDATE_FAILURE_THRESHOLD="${TWIN_UPDATE_FAILURE_THRESHOLD:-00:01:00}"
 METRICS_SCRAPE_FREQUENCY_IN_SECS="${METRICS_SCRAPE_FREQUENCY_IN_SECS:-300}"
 METRICS_UPLOAD_TARGET="${METRICS_UPLOAD_TARGET:-AzureLogAnalytics}"
 
 if [[ "${TEST_NAME,,}" == "longhaul" ]] ||
    [[ "${TEST_NAME,,}" == "stress"  ]]; then
     tracking_id=$(cat /proc/sys/kernel/random/uuid)
     TEST_INFO="$TEST_INFO,TestId=$tracking_id"
+
+    TWIN_UPDATE_FAILURE_THRESHOLD="${TWIN_UPDATE_FAILURE_THRESHOLD:-00:01:00}"
+    DESIRED_PROPERTY_CALLBACK_FAILURE_THRESHOLD="${DESIRED_PROPERTY_CALLBACK_FAILURE_THRESHOLD:-00:01:00}"
```

Comment:
This is just rendering weirdly on laptop screen sizes. The indentation is correct.